platform-core-module/src/main/java/com/softicar/platform/core/module/web/service/WebServiceBrokerServiceExceptionHandler.java

```diff
@@ -0,0 +1,48 @@
+package com.softicar.platform.core.module.web.service;
+
+import com.softicar.platform.common.core.utils.CastUtils;
+import com.softicar.platform.common.core.utils.DevNull;
+import com.softicar.platform.common.network.http.HttpStatusCode;
+import com.softicar.platform.common.network.http.error.HttpError;
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+
+public class WebServiceBrokerServiceExceptionHandler {
```

Comment:
Neither this class nor `WebServiceBrokerService` (in which this class is used) is covered by a unit test. We should fix that.